app/views/partials/_search_bar_sell.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="m-2 w-50 align-self-center">
+  <div class="input-group w-75"> 
+    <%= form_tag seller_path, method: :get, class: "input-group w-100" do %>
+      <%= text_field_tag :query, nil, class: "form-control", placeholder: "Buscar telefono" %>
+      <%= submit_tag "Buscar", class: "btn button-blue" %>
+    <% end %>
+  </div>
+  
+  <% puts "Entering the if block" %>
+  <% if @sell.present? %>
+  <% puts "Entering the if block" %>
+    <ul class="m-1 list-inline w-75">
+      <% @sell.each do |result| %>
+        <li class="p-2 bg-opacity-10 border rounded w-75 list-inline-item align-middle">
+        <%= puts result[0]%>
+        <a> Producto<%= result[1] %> - <%= result[0] %></a>
+        </li>
```

Comment:
indented correctly these lines, and can you explain to me what is the reason to have some `puts` lines?

Reply:
I wasn't getting any response from the controller and was looking for the error, but I already fixed it.